examples/dt/uni-alpha/README.md

```diff
@@ -0,0 +1,154 @@
+# Deployed Topology - Alpha
+
+This document contains a list of integration test suites that would be
+executed against the below specified topology of Red Hat OpenStack Services
+on OpenShift. It also contains a collection of custom resources (CRs) for
+deploying the test environment.
+
+## Purpose
+
+This topology is used for executing integration tests that evaluate the
+`default` backends of the below mentioned services.
+
+## Environment
+
+### Nodes
+
+| Role | Machine Type | Count |
+| ---- | ------------ | ----- |
+| Compact OpenShift | vm | 3 |
+| OpenStack Compute | vm | 2 |
+| Networker         | vm | 3 |
+| Test nodes        | vm | 2 |
+
+### Networks
+
+| Name | Type | Interface |
+| ---- | ---- | --------- |
+| Provisioning | untagged | nic1 |
+| Machine | untagged | nic2 |
+| RH OSP | trunk | nic3 |
+
+#### Networks in RH OSP
+
+| Name | Type |
+| ---- | ---- |
+| ctlplane | untagged |
+| internalapi | VLAN tagged |
+| Storage | VLAN tagged |
+| Tenant | VLAN tagged |
+
+### Services, enabled features and configurations
+
+| Service          | configuration    | Lock-in coverage?  |
+| ---------------- | ---------------- | ------------------ |
+| Cinder           | LVM/iSCSI/lioadm | Must have          |
+| Cinder Backup    | Swift            | Must have          |
+| Glance           | Swift            | Must have          |
+| Swift            | (default)        | Must have          |
+| Octavia          | (amphora)        | Must have          |
+| Horizon          | N/A              | Must have          |
+| Barbican         | (default)        | Must have          |
+| Telemetry        |                  | Must have          |
+| Ironic           |                  | Must have          |
+| Neutron          | OVN - AZs        | Must have          |
+
+#### Support services
+
+The following table lists services which are not the main focus of the testing
+(which may be covered by additional scenarios), but are required for the DT to
+work properly and can be deployed with any/default configuration.
+
+| Service          | Reason            |
+| ---------------- |------------------ |
+| Nova             | needed by scenario testing |
+| Keystone         | needed by all services     |
+
+### Additional configuration
+
+- Always-on, default services and features: TLSe
+- Two additional fake baremetal nodes
+- Availability zones for OVN (zone-1 & zone-2)
+- Logical volume with the name cinder-volumes exists on a OpenShift node.
```

Comment:
This should be covered in more detail, similar to how you discuss iSCSI (L76) and multipath (L101).

For reference, see the latest version of https://github.com/openstack-k8s-operators/cinder-operator/tree/main/config/samples/backends/lvm

It describes the need to add the "openstack.org/cinder-lvm=" label to one (and only one) of the OCP worker nodes and install the LVM MachineConfig at https://github.com/openstack-k8s-operators/cinder-operator/blob/main/config/samples/backends/bases/lvm/lvm.yaml

Reply:
Just to clarify, you'll apply the MachineConfig to all worker nodes, but it relies on the "openstack.org/cinder-lvm=" label to specify the actual node where you want the LVM data to live.

Reply:
@ASBishop thank you for taking the time to review and provide valuable feedback.

Done. I have added information in the latest commit.